packages/contact-manager/README.md

```diff
@@ -0,0 +1,200 @@
+<!--suppress HtmlDeprecatedAttribute -->
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>Contact Manager (Typescript) 
+  <br>
+</h1>
+
+---
+
+**Warning: This package still is in very early development. Breaking changes without notice will happen at this point!**
+
+---
+
+A Veramo contact manager plugin. This plugin manages contacts and identity configurations to third parties and persists them. These configurations can then be used to establish a connection.
+
+## Supported identity connection types
+
+For now the following connection types are supported:
+
+- OpenID
```

Comment:
Is this still relevant. Can we rename these top OpenID Connect and Self Issued OpenID v2 instead of DID Auth SIOP

Reply:
renamed